.github/workflows/security.yml

```diff
@@ -0,0 +1,15 @@
+name: Security analysis
```

Comment:
We are going to combine these into one workflow now, like https://github.com/dvsa/mot-logger/pull/1/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f